bigtop-packages/src/common/kafka/install_kafka.sh

```diff
@@ -164,6 +168,9 @@ ln -s /var/log/kafka ${PREFIX}/$LIB_DIR/logs
 rm -f ${PREFIX}/${LIB_DIR}/libs/zookeeper-*.jar
 ln -sf /usr/lib/zookeeper/zookeeper.jar ${PREFIX}/${LIB_DIR}/libs/
 
+# BIGTOP-3712
+ln -s /usr/lib/zookeeper/zookeeper-jute-${ZOOKEEPER_VERSION}.jar ${PREFIX}/${LIB_DIR}/libs/
```

Comment:
How about making zookeeper-jute.jar as symlink to zookeeper-jute-(VERSION).jar [as done for zookeeper.jar in zookeeper side](https://github.com/apache/bigtop/blob/be8a42bb396dcb75c9fa25d5104ad71eefb409cd/bigtop-packages/src/common/zookeeper/install_zookeeper.sh#L122
).  Then we can create `/usr/lib/kafka/libs/zookeeper-jute.jar` as symlink to `/usr/lib/zookeeper/zookeeper-jute.jar`. 
Those symlinks are meant to work even if ZooKeeper is upgraded from 3.5.9.